src/cephadm/tox.ini

```diff
@@ -49,7 +49,7 @@ deps =
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm.py cephadmlib}
-    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 11'
+    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 7'
```

Comment:
This still references `docker.io`. Should this change?

What was the original point of this test?

Reply:
The original idea was just for us to be keeping track of how many references to docker.io are made within the code. That's it I believe. It has no other purpose.

Reply:
iirc main idea was to avoid having some PR inadvertently adding some docker images without adding them to the constants.py. Do whatever you want with this check.